deploy/operator_dev_cluster.yaml

```diff
@@ -0,0 +1,93 @@
+apiVersion: image.openshift.io/v1
+kind: ImageStream
```

Comment:
aren't we supposed to have a `DeploymentConfig` that uses this `ImageStream`?

Reply:
we are using it here https://github.com/codeready-toolchain/host-operator/pull/38/files#diff-caad408dc9a5d42c733a0e159f579e64R18

Reply:
ah sorry, I missed that  🤦‍♂